thorlcr/thorutil/thmem.cpp

```diff
@@ -262,12 +262,15 @@ class CSharedSpillableRowSet : public CSpillableStreamBase, implements IInterfac
             }
             else if (pos == owner->rows.numCommitted())
             {
-                owner->clearSpillingCallback();
```

Comment:
It was wrong, because you want it to continue to be able to spill (the owner), and not stop just because this stream is at it's end.